packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReactNativeUtils.java

```diff
@@ -27,6 +27,7 @@ public BorderRadii(
   public static BorderRadii getBorderRadii(View view) {
     if (view.getBackground() != null) {
       return BorderRadiiDrawableUtils.getBorderRadii(view);
+      // The rest can be safely removed once support for version 0.75 is dropped.
```

Comment:
Add TODO so it would be easy to locate it.

```suggestion
      // TODO: Remove the rest once support for RN 0.75 is dropped.
```